osu.Game/Overlays/Changelog/UpdateStreamBadge.cs

```diff
@@ -16,99 +14,90 @@
+                    Children = new Drawable[]
                     {
-                        AutoSizeAxes = Axes.X,
-                        RelativeSizeAxes = Axes.Y,
-                        Direction = FillDirection.Vertical,
-                        Children = new[]
+                        text = new FillFlowContainer<SpriteText>
                         {
-                            new OsuSpriteText
-                            {
-                                Text = stream.DisplayName,
-                                Font = OsuFont.GetFont(weight: FontWeight.Bold, size: 12),
-                                Margin = new MarginPadding { Top = 6 },
-                            },
-                            new OsuSpriteText
+                            AutoSizeAxes = Axes.X,
+                            RelativeSizeAxes = Axes.Y,
+                            Direction = FillDirection.Vertical,
+                            Margin = new MarginPadding { Top = 6 },
+                            Children = new[]
                             {
-                                Text = stream.LatestBuild.DisplayVersion,
-                                Font = OsuFont.GetFont(weight: FontWeight.Light, size: 16),
-                            },
-                            new OsuSpriteText
-                            {
-                                Text = stream.LatestBuild.Users > 0 ? $"{stream.LatestBuild.Users:N0} {"user".Pluralize(stream.LatestBuild.Users == 1)} online" : null,
-                                Font = OsuFont.GetFont(weight: FontWeight.Regular, size: 10),
-                                Colour = new Color4(203, 164, 218, 255),
-                            },
-                        }
-                    },
-                    expandingBar = new ExpandingBar
-                    {
-                        Anchor = Anchor.TopCentre,
-                        Colour = stream.Colour,
-                        ExpandedSize = 4,
-                        CollapsedSize = 2,
-                        IsCollapsed = true
-                    },
-                }
-            };
+                                new OsuSpriteText
+                                {
+                                    Text = stream.DisplayName,
+                                    Font = OsuFont.GetFont(size: 12, weight: FontWeight.Black),
+                                },
+                                new OsuSpriteText
+                                {
+                                    Text = stream.LatestBuild.DisplayVersion,
+                                    Font = OsuFont.GetFont(size: 16, weight: FontWeight.Regular),
+                                },
+                                new OsuSpriteText
+                                {
+                                    Text = stream.LatestBuild.Users > 0 ? $"{stream.LatestBuild.Users:N0} {"user".Pluralize(stream.LatestBuild.Users == 1)} online" : null,
```

Comment:
Can be simplified quite a bit, leveraging humanizer:
```suggestion
                                    Text = stream.LatestBuild.Users > 0 ? $"{"user".ToQuantity(stream.LatestBuild.Users, "N0")} online" : null,
```